### DISCUSSION TITLE
最新版取消了OLLAMA了吗？心碎了呀。

### DISCUSSION THREAD

**Discussion by linmaobang** [logancyang/obsidian-copilot#603] [General]
最新版取消了OLLAMA了吗？心碎了呀。

**Comment by logancyang** [comment]
没有取消，本地可以用（看下release视频），self-hosted目前有问题，已经修复马上发布2.6.3.

**Comment by linmaobang** [comment]
👌 感谢yang

Logan Yang ***@***.***> 于 2024年9月6日周五 13:38写道：
